### PR TITLE
chore: update packages

### PR DESCRIPTION
**Pull request by olastor**

### CODE REVIEW

File: src/app/models/user.model.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import mongoose from 'mongoose';
 import bcrypt from 'bcrypt-nodejs';
 
 let userSchema = mongoose.Schema({
-  username: { type: String, unique: true, validate: /^[a-zA-Z][a-zA-Z0-9-_\.]{3,20}$/},
+  username: { type: String, unique: true, validate: /^[a-zA-Z][a-zA-Z0-9-_.]{3,20}$/ },
```
Comment (binarious) [inline src/app/models/user.model.js:5]: Why this change? `\.` allows a dot, but `.` allows any char.

**Review by binarious — CHANGES_REQUESTED** [review]

File: src/app/models/user.model.js (hunk 1 above)
Comment (olastor) [reply src/app/models/user.model.js:5]: Not in this case because it's inside of a character class (`[]`) it makes no difference. It was a linting error so I had to change it. See i.e. https://stackoverflow.com/questions/10397968/escape-dot-in-a-regex-range

Comment (binarious) [reply src/app/models/user.model.js:5]: Ah you're right.

**Review by binarious — APPROVED** [review]
